egs/mini_librispeech/s5/local/chain/tuning/run_tdnn_1c_discriminative.sh

```diff
@@ -0,0 +1,221 @@
+#!/bin/bash
+
+set -o pipefail
+set -e
+# this is run_discriminative.sh
+
+# This script does discriminative training on top of chain nnet3 system.
+# note: this relies on having a cluster that has plenty of CPUs as well as GPUs,
+# the lattice generation runs in about real-time
+
+stage=0
+train_stage=-10 # can be used to start training in the middle.
+get_egs_stage=-10
+use_gpu=true  # for training
+cleanup=false  # run with --cleanup true --stage 6 to clean up (remove large things like denlats,
+               # alignments and degs).
+
+. cmd.sh
+. ./path.sh
+. ./utils/parse_options.sh
+
+srcdir=exp/chain/tdnn1c_sp
+train_data_dir=data/train_clean_5_sp_hires
+online_ivector_dir=exp/nnet3/ivectors_train_clean_5_sp_hires
+tree_dir=exp/chain/tree_sp
+degs_dir=                     # If provided, will skip the degs directory creation
+lats_dir=                     # If provided, will skip denlats creation
+
+## Objective options
+criterion=smbr
+one_silence_class=true
+
+dir=${srcdir}_${criterion}
+
+## Egs options
+frames_per_eg=150
+frames_overlap_per_eg=30
+
+## Nnet training options
+effective_learning_rate=0.000001
```

Comment:
Yeah, I refer to some scripts for the "effective_learning_rate" option.
In tedlium, it is 1.25*10^-6. In librispeech, it is 1*10^-6. In swbd, it is 1.25*10^-7.
But they are all much bigger than mini_librispeech.
I don't have enough experience for it. So do you have any suggestive value for this option?